### DISCUSSION TITLE
Enterprise accounts schema preview always included in introspection queries

### DISCUSSION THREAD

**Discussion by dmitshur** [community/community#24823] [Enterprise]
<p>It seems there’s currently (as of this post) a problem where the default <a href="https://developer.github.com/v4/guides/intro-to-graphql/#discovering-the-graphql-api" rel="nofollow noopener">introspection query</a> for GitHub GraphQL v4 API includes the <a href="https://developer.github.com/v4/previews/#enterprise-accounts" rel="nofollow noopener">Enterprise accounts schema preview</a>, even when it’s not asked for.</p>
<p>Consider:</p>
<p>$ curl -s -H “Authorization: bearer $GITHUB_TOKEN” <a href="https://api.github.com/graphql" target="_blank" rel="nofollow noopener">https://api.github.com/graphql</a> &gt; default.json<br>
$ curl -s -H “Authorization: bearer $GITHUB_TOKEN” -H “Accept: application/vnd.github.enterprise-accounts-preview+json” <a href="https://api.github.com/graphql" target="_blank" rel="nofollow noopener">https://api.github.com/graphql</a> &gt; with_enterprise_preview.json<br>
$ diff ./default.json ./with_enterprise_preview.json <br>
$ echo $?<br>
0</p>
<p>Hope this is helpful to you.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for letting us know! I’ve passed this along to the correct team for investigation. I can’t promise anything or give an ETA, but it’s in the right hands.</p>
<p>Thanks again :+1:</p>

**Comment by dmitshur** [comment]
<p>The Enterprise accounts schema preview has been merged into the stable API by now, so this is resolved.</p>
